robot/impl/robot_reconfigure_test.go

```diff
@@ -2745,12 +2745,12 @@ func TestRemoteRobotsGold(t *testing.T) {
 	err = remote3.StartWeb(ctx, options)
 	test.That(t, err, test.ShouldBeNil)
 
-	utils.SelectContextOrWait(ctx, 26*time.Second)
+	utils.SelectContextOrWait(ctx, 5*time.Second)
```

Comment:
I think we have the waits this long because of flake - would be good to run some of these tests 100 times or something to make sure we aren't reintroducing them

Reply:
gotcha; ran all tests locally (M1 mac) 100 times each and saw no failures.